apps/meteor/client/views/admin/moderation/helpers/UserColumn.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box } from '@rocket.chat/fuselage';
+import React from 'react';
+
+import UserAvatar from '../../../../components/avatar/UserAvatar';
+
+type UserColumnProps = {
+	name?: string;
+	username?: string;
+	isDesktopOrLarger?: boolean;
+	isProfile?: boolean;
+};
+
+const UserColumn = ({ name, username, isDesktopOrLarger, isProfile }: UserColumnProps) => {
```

Comment:
try to avoid props starting with `is` usually is should be used in functions

also usually using boolean props for customize components is a code smell , probably you are trying to save sometime here instead of using 2..3 different compoentns

Reply:
https://www.informit.com/articles/article.aspx?p=1392524

“Boolean arguments loudly declare that the function does more than one thing. They are confusing and should be eliminated.”
— Clean Code, Robert C. Martin (Uncle Bob)

Reply:
Thank you! This was a good read. 

I have changed the props to accept the `size` so I could use the component, instead of creating multiple of them.